docs/feature-gates.md

```diff
@@ -138,3 +139,28 @@ foo           bar                   1          12        1221    2020-09-07T13:2
 #### Requirements for this Feature
 
 None
+
+### NodePortLocal
+
+`NodePortLocal` is a feature that runs as part of the Antrea Agent, through which
+each port of a Pod can be reached from external network using a port in the Node
+on which the Pod is running. In addition to enabling NodePortLocal feature gate,
+the value of `nplPortRange` must be set in Antrea Agent configuration through ConfigMap.
```

Comment:
"must"? I thought it would default to "40000-41000"

Reply:
Wanted the user to be aware of the range, but yes, this not a must. Updated.